pkg/version/version.go

```diff
@@ -23,7 +23,7 @@ import (
 // Version returns the version of the main module
 func Version() string {
 	info, ok := debug.ReadBuildInfo()
-	if !ok {
+	if !ok || (info != nil && info.Main.Version == "") {
```

Comment:
A unit test failed without this change

It looks like previously ok was false. Now we get ok==true + an info object with an empty Version

Reply:
okay, interesting. That is IMHO a go bug as it breaks comoatibility.

Reply:
Shouldn't it be
```suggestion
	if !ok || info == nil || info.Main.Version == "" {
```

Reply:
see #662 for explanation for this issue